scripts/end-to-end.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright (c) Mainflux
+# SPDX-License-Identifier: Apache-2.0
+
+###
+# Runs all Mainflux microservices (builds and installs if not done already)
+# 
+# Uses Schemathesis to check the openAPI configuration with the actual endpoints
+###
+
+cd ..
+
+make dockers
```

Comment:
Make dockers in dev mode, like:
`make -j $NUMPROC` and `make -j $NUMPROC dockers_dev` where `$NUMPROC` is the number of processing cores available on CI (check how to get that).

Reply:
it has 2 cores, used lscpu command